tests/integration/targets/azure_rm_openshiftmanagedcluster/tasks/main.yml

```
@@ -58,6 +58,19 @@
     address_prefix_cidr: 10.152.0.0/24
   register: worker_sub_output
 
+- name: Get available openshift cluster versions
+  azure_rm_openshiftmanagedclusterversion_info:
+    location: "eastus"
+  register: ocp_cl_vers
+
+- name: Extract the latest available version
+  set_fact:
```

Use `ansible.builtin.set_fact` or `ansible.legacy.set_fact` instead
```suggestion
  ansible.builtin.set_fact:
```

---

Done.